script_runner.py

```diff
@@ -78,6 +79,9 @@ def _run_command(self, cmd, working_dir):
 
         command_thread.start()
 
+        console_flush_thread = threading.Thread(target=self.__trigger_flush_console_output)
```

Comment:
EOD: Initiating __trigger_flush_console ```Timer``` now. However, the build process is now failing. Looking into it.